snowflake/objects/deploy.sql

```diff
@@ -0,0 +1,77 @@
+/*
+  This is the primary deployment script for RECOVER Snowflake objects.
+
+  Deployments are made to a specific environment. An environment is analogous
+  to a database. Account-level objects are not covered by this script. We assume
+  that the following objects have already been created at the account level:
+
+    - ROLE `RECOVER_DATA_ENGINEER`
+        * A ROLE with the appropriate privelages to create and deploy
+          objects in a database.
+    - WAREHOUSE `RECOVER_XSMALL`
+    - API INTEGRATION `RECOVER_GIT`
+        * An API INTEGRATION with an API_ALLOWED_PREFIXES value
+          containing 'https://github.com/Sage-Bionetworks/' and
+          API_PROVIDER = GIT_HTTPS_API.
+    - STORAGE INTEGRATION `RECOVER_PROD_S3`
```

Comment:
Do we also want a `recover_dev_s3`? Not sure if we want non-releases to go through any of this.

Reply:
We actually do already have a storage integration for our dev account. But it's not a dependency for this deployment, and since in the future we will clone prod tables (or maybe we want to sample them to keep things lightweight?) to their own isolated environment I don't see why we would want to work with the pilot data.